examples/browser/locator_filter.js

```diff
@@ -35,7 +35,7 @@ export default async function() {
 
   // Filter the 2nd item by filtering items with the "Product 2" text.
   await check(page, {
-    'textContent': async p => {
+    'Use locator.filter the Add to cart 2 button': async p => {
```

Comment:
```suggestion
    'Use locator.filter to filter the Add to cart 2 button': async p => {
```
Nit: I know this is a bit redundant but the initial sentence confuses me a bit :sweat_smile:

Reply:
Ah, right :)